docs/using-the-jdbc-driver/using-plugins/UsingTheDeveloperPlugin.md

```diff
@@ -0,0 +1,43 @@
+## Developer Plugin
+
+The Developer Plugin allows developers to inject a prepared exception to a connection, let the connection to raise this exception on the next call and let an application to process it. Some of the exception that the driver may raise is quite hard to get and it's by the nature of such exceptions. For example, network issues may cause the driver to raise various timeout exceptions. However it may require substantial efforts to design and to build a testing environment where such timeout exceptions could be produced with 100% accuracy and 100% guarantee. If test suite can't verify such network outages cases with 100% accuracy it significantly decreases the value of such tests and makes the tests flaky. The Developer Plugins simplifies testing of such scenarios as shown below.
```

Comment:
Big paragraph. I would suggest splitting it in two. (Just breaking it adding a space line somewhere).

Ideally, i would start with one sentence with a simple overview of what it does, then continuing with the description.

```
The Developer Plugin is a plugin that allows developers to inject a custom exception to a connection for testing/debugging purposes, and see how the application will handle it.
[space]
Since some exceptions raised by the drivers happen only rarely/seldomly, testing for those might be difficult or require a lot of effort in building a testing environment. Classic examples of those are network outages. Etc...
```